kotlin/workflow-ui/core-android/src/main/java/com/squareup/workflow/ui/WorkflowLayout.kt

```diff
@@ -41,13 +40,17 @@ import kotlinx.coroutines.flow.onEach
 class WorkflowLayout(
   context: Context,
   attributeSet: AttributeSet? = null
-) : FrameLayout(context, attributeSet) {
+) : ViewGroup(context, attributeSet) {
   private val showing: WorkflowViewStub = WorkflowViewStub(context).also {
-    addView(it, ViewGroup.LayoutParams(MATCH_PARENT, MATCH_PARENT))
+    addView(it, LayoutParams(MATCH_PARENT, MATCH_PARENT))
   }
 
   private var restoredChildState: SparseArray<Parcelable>? = null
 
+  private val child: View
+    get() = getChildAt(0)
+        ?: error("Child must be set immediately upon creation.")
```

Comment:
@rjrjr should confirm, but I think this can be more safely expressed as `showing.actual ?: showing`.

The view is always added to the hierarchy, so there's no need to validate that here.